.github/CODEOWNERS

```diff
@@ -1,5 +1,9 @@
 # Project Owners
+<<<<<<< HEAD
```

Comment:
This appears to be a merge conflict committed.

Reply:
Yeah I figured there is some merge conflicts which were commited and that should not, I'll make an update of the merge request after rechecking all the files and fixing the conflicts not handled